### ISSUE TITLE
NgOptimizedImage: unclear behavior when ngSrcset is used with default image loader

### ISSUE THREAD

**Issue by Timbo1979** [angular/angular#48655]
### Which @angular/* package(s) are the source of the bug?

common

### Is this a regression?

Yes

### Description

In my Angular 15.x project I'm using NgOptimizedImage with ngSrc and ngSrcset but getting incorrect result according to the documentation.

When I enter:

`<img ngSrc="hello.jpg" ngSrcset="100w, 200w" />`

Documentation says I should get:
`<img src="path/hello.jpg" srcset="path/hello.jpg?w=100 100w, path/hello.jpg?w=200 200w" />`

Instead I get:
`<img src="path/hello.jpg" srcset="path/hello.jpg 100w, path/hello.jpg 200w" />`

So the same image is returned for all viewports.

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/angular-bf98bj?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.module.ts

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI: 15.0.1
Node: 16.15.1
Package Manager: npm 9.1.2
OS: win32 x64

Angular: 15.0.1
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1500.1
@angular-devkit/build-angular   15.0.1
@angular-devkit/core            15.0.1
@angular-devkit/schematics      15.0.1
@schematics/angular             15.0.1
rxjs                            7.5.7
typescript                      4.8.4
```

**Comment by jnizet** [comment]
Hi @Timbo1979 

As far as I understand, the URL that is generated depends on the ImageLoader that you have provided. If you haven't provided any, then the default one will just return the path of the image, regardless of the requested width, because that's what allows loading a static image.

**Comment by AndrewKushnir** [comment]
@Timbo1979 as @jnizet mentioned above, there is no image loader configured ([see docs](https://angular.io/guide/image-directive#configuring-an-image-loader-for-ngoptimizedimage)), thus there were no image URL transformation performed.

I'm leaving this ticket open to see if we should add a warning when the `ngSrcset` is present and a default image loader is used.

**Comment by AndrewKushnir** [comment]
We've discussed this topic with the team and we'd like to add a logic to output a warning in case the `ngSrcset` is present and no loaders are configured (i.e. only the default one is present). The warning should be helpful for cases like the one described in this ticket.

I'm adding the "help wanted" label to see if there is an interest from community members to contribute a PR with this change.

Thank you.

**Comment by atcastle** [comment]
@AndrewKushnir Let's just close this issue. It's fixed by #48804
